From 4dc6747be00570617ffa359a5694b2b70c8f08dc Mon Sep 17 00:00:00 2001
From: Pierre-Yves <pyu@riseup.net>
Date: Wed, 21 Nov 2018 22:40:17 +0100
Subject: [PATCH] Fix docbook versions/paths

Signed-off-by: Pierre-Yves <pyu@riseup.net>
---
 po4a/Makefile    | 4 ++--
 scripts/Makefile | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/po4a/Makefile b/po4a/Makefile
index e2b1258..1b815a6 100644
--- a/po4a/Makefile
+++ b/po4a/Makefile
@@ -28,7 +28,7 @@ de/%.de.1: de/%.de.pod translate
 	pod2man --utf8 --center=" " --release="Debian-Hilfswerkzeuge" $< > $@
 de/%.de.1: de/%.de.dbk translate
 	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	  /usr/share/xml/docbook/xsl-stylesheets-1.79.2/manpages/docbook.xsl $<
 # xsltproc writes out to foo.1 even if you tell it to do otherwise, so we have
 # to rename the file after it's generated
 	mv de/$*.1 $@
@@ -41,7 +41,7 @@ fr/%.fr.1: fr/%.fr.pod translate
 	pod2man --utf8 --center=" " --release="Utilitaires Debian" $< > $@
 fr/%.fr.1: fr/%.fr.dbk translate
 	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	  /usr/share/xml/docbook/xsl-stylesheets-1.79.2/manpages/docbook.xsl $<
 # xsltproc writes out to foo.1 even if you tell it to do otherwise, so we have
 # to rename the file after it's generated
 	mv fr/$*.1 $@
diff --git a/scripts/Makefile b/scripts/Makefile
index f8ac7a0..dcb1983 100644
--- a/scripts/Makefile
+++ b/scripts/Makefile
@@ -82,7 +82,7 @@ endif
 	pod2man --utf8 --center=" " --release="Debian Utilities" $< > $@
 %.1: %.dbk
 	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	  /usr/share/xml/docbook/xsl-stylesheets-1.79.2/manpages/docbook.xsl $<
 
 # Syntax checker
 test_sh: $(SH_CHECKS)
@@ -121,7 +121,7 @@ clean:
 test: test_pl test_sh test_py
 
 install: all
-	python3 setup.py install --root="$(DESTDIR)" --no-compile --install-layout=deb
+	python3 setup.py install --root="$(DESTDIR)" --no-compile
 	cp $(SCRIPTS) $(DESTDIR)$(BINDIR)
 	ln -sf edit-patch $(DESTDIR)$(BINDIR)/add-patch
 	install -d $(DESTDIR)$(COMPL_DIR)
-- 
2.19.2

